Pdf4QtViewer/pdfsendmail.cpp

```diff
@@ -84,6 +84,9 @@ bool PDFSendMail::sendMail(QWidget* parent, QString subject, QString fileName)
             return false;
     }
 
+    return false;
+#elif defined(Q_OS_UNIX)
+    // TODO
```

Comment:
Unix implementation here is not so important, so issue can be created for this...